squad/api/rest.py

```diff
@@ -571,6 +572,32 @@ def unsubscribe(self, request, pk=None):
         data = {"email": subscriber_email}
         return Response(data, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['get', 'post'], suffix='compare_builds')
+    def compare_builds(self, request, pk=None):
+        builds_to_compare = {
+            "baseline": request.data.get("baseline", None) if request.method == 'POST' else request.query_params.get("baseline", None),
+            "to_compare": request.data.get("to_compare", None) if request.method == 'POST' else request.query_params.get("to_compare", None)
+        }
+        if any(builds_to_compare.values()) and all(builds_to_compare.values()):
```

Comment:
The `any()` doesn't enforce that all values in the dictionary are filled, but `all` does, I think it's safe to remove the `any` then

Reply:
@chaws the problem is all() yields true if both values are None as well and when combined with any(), which will returns false in this case (both empty) then this corner is handled properly. Think we should just abort in case both values are empty?

Reply:
I was wrong. I thought if endpoint was called without any args at all, then `builds_to_compare` will be an empty dict without even keys. Totally got myself confused :x: